sourcegraph/jetbrains
# Report all agent errors using the chat or status bar

Fixes: Invalid access token reported
[#289](https://github.com/sourcegraph/jetbrains/issues/289)
[#762](https://github.com/sourcegraph/jetbrains/issues/762)

Now every chat or agent error is properly reported on the UI:

![image](https://github.com/sourcegraph/jetbrains/assets/1519649/b6cbfb03-fc51-4ffc-9747-339884e7e413)


## Test plan

1. Create new access token
2. Login using that access token
3. Run some Cody actions, like chat question and autocomplete
4. Remove access token from SG instance
5. Run some Cody action and check if it properly reports an error
6. Switch account and check if error is gone
7. Switch account back and check if error is reported again
